migrations/057-arbitrum-all-current-votes.sql

```diff
@@ -0,0 +1,52 @@
+DROP FUNCTION IF EXISTS api.all_current_votes(CHAR);
+
+--testing to replace 38
+CREATE OR REPLACE FUNCTION api.all_current_votes(arg_address CHAR)
+RETURNS TABLE (
+	poll_id integer,
+	option_id_raw character,
+	option_id integer,
```

Comment:
@b-pmcg  can you allow to return all the votes without grouping by pollId? 

I want to have all the history of votes and edits of a user, so in the portal we can limit for how many times a user voted on a poll, and also show the history